source/electrical_heating/src/electrical_heating/domain/calculation.py

```diff
@@ -240,8 +242,34 @@ def execute_core_logic(
         .alias("quantity"),
     ).drop_duplicates()
 
-    period_consumption_with_limit = convert_timezone(
-        period_consumption_with_limit, time_zone, to_utc=True
+    compare_previous_calculated_daily_consumption = (
```

Comment:
"compare" is imperative. Suitable for functions that does something. The data frame holds data that has changed (compared to previous)
```suggestion
    changed_consumption = (
```